daemon/src/db.rs

```diff
@@ -49,6 +49,8 @@ pub fn connect(path: PathBuf) -> BoxFuture<'static, Result<SqlitePool>> {
             return connect(path).await;
         }
 
+        tracing::info!("Opened database at {}", path.display());
```

Comment:
🔧 Can probably squash this with the previous commit, unless there was a reason for it.